### DISCUSSION TITLE
Tailwindcss v3 not updating changes in dev - Gridsome

### DISCUSSION THREAD

**Discussion by paragchirde** [tailwindlabs/tailwindcss#7020] [Help]
I just upgraded my project to **Tailwind v3** and ran it on M1 (pro) mac. Only if I am saving the _`tailwind.config.js`_ file or restarting the project, it reflects the change.
I also tried doing the follows:
A. Add a .env file with `TAILWIND_MODE=watch` 
AND
B. adding the same in my dev script
`gridsome develop TAILWIND_MODE=watch`

Although this worked on my intel based mac but the same project is not working on new M1 mac. 

Little help needed to resolve this. 

Sample Repo
[https://github.com/paragchirde/Tailwind-v3-gridsome-sample-repo](url)

**Comment by EricHier** [comment]
Hi @paragchirde ! Did you find a solution? :) I just stumbled across the same problem.

**Comment by arnoldski** [comment]
I'm experiencing the same issue, any updates @paragchirde?
